projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-details/quote-details.component.html

```diff
@@ -181,14 +233,18 @@ <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
             </div>
           </li>
           <li class="accordion-list-item row">
-            <div class="col-6 col-sm-5 item-label">
-              {{ 'fscommon.paymentMethod' | cxTranslate }}
-            </div>
-            <div class="col-6 col-sm-5 value notice">
-              <ng-container>
-                {{ cart.paymentType?.displayName }}
+            <ng-container *ngIf="cart.paymentInfo">
+              <ng-container
+                *ngIf="cart.paymentInfo.cardNumber; else defaultPayment"
+              >
+                <div class="col-6 col-sm-5 item-label">
+                  {{ 'fscommon.paymentMethod' | cxTranslate }}
+                </div>
+                <div class="col-6 col-sm-5 value notice">
```

Comment:
since this is the only difference maybe we don't need `defaultPayment` template, instead:
 ```
{{ cart.paymentInfo.cardNumber ? cart.paymentType?.displayName : 'fscommon.invoice' | cxTranslate  }}
```

Reply:
Agree, it is definitely better solution. Done!